hack/lib/util/environment.sh

```diff
@@ -145,12 +145,14 @@ function os::util::environment::setup_tmpdir_vars() {
     HOME="${FAKE_HOME_DIR}"
     export HOME
 
-    # ensure that the directories are clean
-    for target in $( ${USE_SUDO:+sudo} findmnt --output TARGET --list ); do
-        if [[ "${target}" == "${BASETMPDIR}"* ]]; then
-            ${USE_SUDO:+sudo} umount "${target}"
-        fi
-    done
+		if [[ "$(uname)" != "Darwin" ]]; then
```

Comment:
I like that logic. Can we add it as a comment to this `if` statement and then merge this?